PowerUp/app/src/main/java/powerup/systers/com/aboutpage.java

```diff
@@ -0,0 +1,27 @@
+package powerup.systers.com;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.media.Image;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.ImageButton;
+
+/**
+ * Created by Medha on 01-11-2015.
+ */
+public class aboutpage extends Activity {
```

Comment:
Class names should be in CamelCase, so in this case that should be `AboutActivity`. Name activities with names including `Activity`so it's easier to know what they are